src/test/kotlin/kt_junit5_test.bzl

```diff
@@ -1,7 +1,7 @@
 load("//kotlin:jvm.bzl", _kt_jvm_test = "kt_jvm_test")
 
 def kt_jvm_test(name, args = [], deps = [], runtime_deps = [], **kwargs):
-    test_package = native.package_name().replace("src/test/kotlin", "").replace("/", ".")
+    test_package = native.package_name().replace("src/test/kotlin/", "").replace("/", ".")
```

Comment:
Probably worth while adding a test case for this package parsing.

Reply:
Ayup.